ipaclient/setup.py

```diff
@@ -56,5 +63,6 @@
         extras_require={
             "install": ["ipaplatform"],
             "otptoken_yubikey": ["yubico", "usb"]
-        }
+        },
+        zip_safe=False,
```

Comment:
Ah, Jinja uses a pkg_resources ResourceManager to access files. So in theory, zip loading would work.

In practice the option and feature is longer relevant. Python eggs are a dying species and Python wheels no longer use zip deployment.